scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -435,3 +361,46 @@ task GetSampleIds {
     File gvs_ids = "gvs_ids"
   }
 }
+
+task CurateInputLists {
+  input {
+    String dataset_name
+    String project_id
+    File input_vcf_index_list
+    File input_vcf_list
+    File input_samples_to_be_loaded_map
+    File input_sample_name_list
+
+    String? service_account_json_path
+  }
+
+  String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+  command <<<
+    set -ex
+    if [ ~{has_service_account_file} = 'true' ]; then
+      gsutil cp ~{service_account_json_path} local.service_account.json
+      gcloud auth activate-service-account --key-file=local.service_account.json
+    fi
+
+    gsutil cp  ~{input_vcf_index_list} input_vcf_index_list_file
```

Comment:
Aren't these 4 files all localized by cromwell? I don't understand why you need to gsutil cp them here.

Reply:
right you are — fixed